src/offchainapi/tests/test_bech32.py

```diff
@@ -20,48 +20,48 @@ def test_bech32_valid_address() -> None:
 
     # do not set a subaddress
     bech32_libra_address = bech32_address_encode(
-        LBR, some_address, none_sub_address
+        DM, some_address, none_sub_address
     )
-    assert bech32_libra_address == "lbr1p7ujcndcl7nudzwt8fglhx6wxnvqqqqqqqqqqqqqflf8ma"
+    assert bech32_libra_address == "dm1p7ujcndcl7nudzwt8fglhx6wxnvqqqqqqqqqqqqqflf8ma"
 
-    hrp, version, address, subaddress = bech32_address_decode(bech32_libra_address, LBR)
-    assert hrp == LBR
+    hrp, version, address, subaddress = bech32_address_decode(bech32_libra_address, DM)
+    assert hrp == DM
     assert version == 1
     assert address == some_address
     assert subaddress == zero_sub_address
 
     # set subaddress to all zeros and check we get the same as if we never added one
     bech32_libra_address = bech32_address_encode(
-        LBR, some_address, zero_sub_address
+        DM, some_address, zero_sub_address
     )
-    assert bech32_libra_address == "lbr1p7ujcndcl7nudzwt8fglhx6wxnvqqqqqqqqqqqqqflf8ma"
+    assert bech32_libra_address == "dm1p7ujcndcl7nudzwt8fglhx6wxnvqqqqqqqqqqqqqflf8ma"
 
     # set some subaddress
     some_sub_address = bytes(bytearray.fromhex("cf64428bdeb62af2"))
 
     bech32_libra_address = bech32_address_encode(
-        LBR, some_address, some_sub_address
+        DM, some_address, some_sub_address
     )
-    assert bech32_libra_address == "lbr1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t"
+    assert bech32_libra_address == "dm1p7ujcndcl7nudzwt8fglhx6wxn08kgs5tm6mz4usw5p72t"
```

Comment:
did you run all tests? only change lbr to dm won't work, as the checksum will also change